docs/RFCS/sql_privileges.md

```diff
@@ -0,0 +1,197 @@
+	`CREATE` privilege on table.
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-createtable.html):
+	`CREATE` privilege in schema.
+* cockroach: `CREATE` privilege on database.
+
+`DROP DATABASE`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/drop-database.html):
+	`DROP` on database.
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-dropdatabase.html):
+	database owner only.
+* cockroach: `DROP` on database, or `root` user.
+
+`DROP TABLE`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/drop-table.html):
+	`DROP` on table.
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-droptable.html):
+	table or schema owner. or `SUPERUSER`.
+* cockroach: `DROP` on table.
+
+`GRANT`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/grant.html):
+	`GRANT OPTION` and the granted privileges.
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-grant.html):
+	`GRANT OPTION` and the granted privileges.
+* cockroach: `GRANT` on affected object. Does not require the granted privileges.
+
+`REVOKE`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/revoke.html):
+	`GRANT OPTION` and the revoked privileges (another invocation has stricter requirements).
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-revoke.html):
+	owner, or `GRANT OPTION` on revoked privileges. Complex rules for grantees.
+* cockroach: `GRANT` on affected object.
+
+`SELECT`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/select.html):
+  `SELECT` (doc seems incomplete).
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-select.html):
+  `SELECT` if tables are read. ALso requires `UPDATE` when using `FOR UPDATE`.
+* cockroach: `SELECT` if tables are read.
+
+`INSERT`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/insert.html):
+  `INSERT`. `UPDATE` or `SELECT` for `ON DUPLICATE KEY UPDATE`.
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-insert.html):
+  `INSER`. `SELECT` if using `RETURNING`.
+* cockroach: `INSERT` on table. `SELECT` if using `SELECT` clause on tables.
+
+`DELETE`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/delete.html):
+	`DELETE` on table. `SELECT` if `WHERE` clause operates on tables.
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-delete.html):
+	`DELETE` on table. `SELECT` if `USING` or `WHERE` clauses operate on tables.
+* cockroach: `DELETE` on table. `SELECT` if `WHERE` clause operates on tables.
+
+`UPDATE`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/update.html):
+	`UPDATE`. Also `SELECT` for `WHERE` clause with table.
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-update.html):
+	`UPDATE`. Also `SELECT` for `WHERE` clause with table.
+* cockroach: `UPDATE`. Also `SELECT` for `WHERE` clause with table.
+
+`TRUNCATE`:
+* [mysql](https://dev.mysql.com/doc/refman/5.7/en/truncate-table.html):
+	`DROP` on table.
+* [postgres](http://www.postgresql.org/docs/9.4/static/sql-truncate.html):
+  `TRUNCATE` privilege.
+* cockroach: `DROP` on table.
```

Comment:
mysql uses `DROP` since they technically do a `DROP` followed by a `CREATE`.
Logically, `TRUNCATE` is also much close to that since it does not do any of the delete logic (no scan, no cascading, etc..).